data/campaigns/World_Conquest/resources/strings/effects.cfg

```diff
@@ -82,7 +82,7 @@ _ "terrain^water" #enddef
 _ "improved defenses" #enddef
 
 #define STR_LIVING
-_ "no undead^consumed by a living beign:" #enddef
+_ "no undead^consumed by a living being:" #enddef
 
 #define STR_UP_TO VALUE
 _ "up to"+" {VALUE}% " #enddef
```

Comment:
The character between "up" and "to" is a non-breaking space, rather than a normal space.

Reply:
Maybe that was deliberate? I don't know. Changed, as requested.

Reply:
See @gfgtdf's remarks below.